clients/secret/secret.go

```diff
@@ -0,0 +1,165 @@
+package secret
+
+import (
+	"context"
+	"fmt"
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.SecretsApi
+	api.OrganizationsApi
+}
+
+type secretPrintOpt struct {
+	deleted bool
+	secret  secret
+	secrets []secret
+}
+
+type secret struct {
+	key   string
+	orgID string
+}
+
+type DeleteParams struct {
+	clients.OrgParams
+	Key string
+}
+
+func (c Client) Delete(ctx context.Context, params *DeleteParams) error {
+	orgID, err := c.getOrgID(ctx, params.OrgParams)
+	if err != nil {
+		return err
+	}
+
+	if err := c.PostOrgsIDSecrets(ctx, orgID).Execute(); err != nil {
```

Comment:
That's an unfortunate name... can you add a comment above this saying that `PostOrgsIDSecrets` actually deletes the secret, and the name is generated from the `operationId` in our API docs?